kubemarine/coredns.py

```diff
@@ -21,8 +21,16 @@
 
 
 def enrich_add_hosts_config(inventory, cluster):
-    if not inventory['services']['coredns']['configmap'].get('Hosts'):
-        inventory['services']['coredns']['configmap']['Hosts'] = system.generate_etc_hosts_config(inventory, cluster)
+    if inventory['services']['coredns']['add_etc_hosts_generated']:
+        if not inventory['services']['coredns']['configmap'].get('Hosts'):
+            inventory['services']['coredns']['configmap']['Hosts'] = system.generate_etc_hosts_generated_config(cluster.inventory, cluster)
+        else:
+            inventory['services']['coredns']['configmap']['Hosts'] += system.generate_etc_hosts_generated_config(cluster.inventory, cluster)
```

Comment:
We stopped adding `localhost` to the `services.coredns.configmap.Hosts`. Was it intended?

Reply:
Actually `localhost` was added to `services.coredns.configmap.Hosts`  only when no custom `services.coredns.configmap.Hosts` were present. 
Now I've added `localhost` to default settings of `services.coredns.configmap.Hosts` so it's added to the configmap in case of a user doesn't override this default setting.